clang/lib/AST/ItaniumMangle.cpp

```diff
@@ -3770,7 +3770,9 @@ void CXXNameMangler::mangleBareFunctionType(const FunctionProtoType *Proto,
 
   if (FD) {
     FunctionTypeDepth.enterResultType();
-    mangleRequiresClause(FD->getTrailingRequiresClause());
+    // FIXME: Should the mangling include the ArgumentPackSubstitutionIndex as
```

Comment:
Is this a `we don't know`, or a `itanium is not clear`?  In the former, we should figure it out (as it seems like we SHOULD?).  In the latter, I'd like an issue on the itanium tracker with an example to figure it out.

Reply:
I will make this part of another PR since we are backing off the subsumption changes for now.